centos2ol_wrapper.sh

```diff
@@ -0,0 +1,407 @@
+#!/bin/bash
+# centos2ol_wrapper -- Wrapper with sanity checks for centos2ol
+# Copyright Nikolai Bezroukov, 2020. Licensed under Perl Artistic license.
+#:: 
+#:: The script accepts options -d, -b -p, -r (other than -h which print this help screen and -v for verbose output)
+#:: You can also adapt configuration parameters that set defaults for those options 
+#:: See  http://www.softpanorama.org/Commercial_linuxes/Oracle_linux/conversion_of_centos_to_oracle_linux.shtml for details
+#:: 
+#:: OPTIONS
+#::
+#::    -b BASE directory -- directory for backup and temp files 
+#:: 
+#::    -v -- produces trace of script execution by setting bash -x option 
+#::
+#::    -r -- change the root password to new value before invoking centos2ol.sh script. 

```

Comment:
This option would not be accepted by Oracle Global Product Security as we do not allow any script to hard-code any user credentials, particularly not root passwords. We should definitely alert folks that they should know the `root` password for all their systems prior to migration.